wdl2cwl/main.py

```diff
@@ -91,9 +93,9 @@ def get_command(
     return new_command
 
 
-def main(argv: List[str]) -> str:
+def convert(args: Namespace) -> str:
```

Comment:
For testing purposes, it would be better if this function just did the conversion (returning a string) and the `main` function handled the file writing

Reply:
Therefore the `convert` function would just accept one argument, the path to the WDL file